index.bs

```diff
@@ -9687,6 +9687,10 @@ from other interfaces. Specifically:
     will correspond to properties on the object itself rather than on
     [=interface prototype objects=].
 
+A [=realm=] can be defined to
+<dfn export for="realm" lt="has a mutable global prototype chain">have a mutable global prototype chain</dfn>.
+If a [=realm=] is not defined as such, the prototype of its global object can not be changed.
```

Comment:
I keep having to learn that conditional fields I have added in the past have led to some kind of issue that required changing them to always-present fields. I would suggest we make this an always-present field of realms with a default of false.

> Each realm has an associated "uses mutable global prototype chain" (a boolean). It is initially false.
>
> Note: This allows the ShadowRealm global to have a mutable prototype.